File: packages/expo-font/android/src/main/java/expo/modules/font/FontLoaderPackage.java
package expo.modules.font;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import expo.modules.core.BasePackage;
import expo.modules.core.ExportedModule;

public class FontLoaderPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.<ExportedModule>singletonList(new FontLoaderModule(context));
  }
}

File: packages/expo-font/android/src/main/java/expo/modules/font/FontLoaderPackage.kt
package expo.modules.font

import android.content.Context
import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class FontLoaderPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf<ExportedModule>(FontLoaderModule(context))
  }
}
